.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Proposed changes
+
+Include description and references to tickets or issues that this PR is related to
+
+## Checklist prior to adding reviewers
+
+_Put an `x` in the boxes that apply._
+
+- [ ] I have run this successfully locally using `./build.ps1 compose`
```

Comment:
Why not implement a version of this in Bash, so that people who don't use Windows don't have to needlessly install PowerShell?